barretenberg/bbup/README.md

```diff
@@ -0,0 +1,54 @@
+# BBup
+
+BBup is a CLI tool that makes it easy to install the [Barretenberg](https://github.com/AztecProtocol/aztec-packages/blob/master/barretenberg/README.md) proving backend.
+
+It assumes you are using [Noir](https://noir-lang.org) as the frontend language.
+
+## Installation
+
+### Dependencies
+
+TODO
+
+### Installation script
+
+BBup is an installer for whatever version of BB you may want. Install BBup with:
+
+```bash
+curl -L bbup.dev | bash
+```
+
+> [!IMPORTANT]
+> *Always* check what scripts do. The above one redirects to [the install script](https://github.com/AztecProtocol/aztec-packages/blob/master/barretenberg/bbup/install) which checks if you have `npm`, installing it with `nvm` otherwise. It then installs [bbup](https://github.com/AztecProtocol/aztec-packages/blob/master/barretenberg/bbup/bbup.ts) globally.
+
+## Usage
+
+To install the Barretenberg version compatible with the current installed Noir version (ex. installed with [noirup](https://github.com/noir-lang/noirup)), run:
+
+```bash
+bbup
+```
```

Comment:
~~This isn't still the case right?~~

Looks like it is but this is pretty janky default behaviour for an installer.

Reply:
Assuming we don't internally require `bbup` to fetch the latest bb version (as in "we always pass -v xx.xx.xx"), I'd rather keep it weird but short for most external devs who simply want to run `noirup && bbup` on new releases